frontend/src/test/scala/bloop/tasks/RunTasksSpec.scala

```diff
@@ -122,23 +124,32 @@ class RunTasksSpec {
   @Test
   def canRunInheritedMainWithForking() = canRunInheritedMain(fork = true)
 
-  private def runAndCheck(sources: Seq[String], fork: Boolean, cmd: Commands.Command)(
-      check: List[(String, String)] => Unit): Unit = {
-    val namedSources = sources.zipWithIndex.map { case (src, idx) => s"src$idx.scala" -> src }.toMap
-    val projectsStructure = Map(projectName -> namedSources)
-    val javaEnv = JavaEnv.default(fork)
-    checkAfterCleanCompilation(projectsStructure, noDependencies, javaEnv = javaEnv, quiet = true) {
-      state =>
-        val recordingLogger = new RecordingLogger
-        val recordingStream = ProcessLogger.toOutputStream(recordingLogger.info _)
-        val recordingState = state.copy(logger = recordingLogger)
-        val project = getProject(projectName, recordingState)
-        val _ = Interpreter.execute(Run(cmd), recordingState)
-        check(recordingLogger.getMessages)
+  @Test
+  def runCanSeeCompileResourcesWithoutForking() = runCanSeeCompileResources(fork = false)
+
+  @Test
+  def runCanSeeCompileResourcesWithForking() = runCanSeeCompileResources(fork = true)
+
+  private def runCanSeeCompileResources(fork: Boolean): Unit = {
+    val projectName = "with-resources"
+    val mainClassName = "Main"
+    val state0 = loadTestProject(projectName)
+    val state =
```

Comment:
For expressions that span for more than one line, can we use blocks to wrap them? It follows the Scalajs guide and I think it's highly desirable for readability purposes.